docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+  web:
+      build: ./
```

Comment:
https://github.com/hitalos/laravel

Reply:
yep? it's not supporting gmp-ext, thatswhy i had to add dockerfile and build: ./

Reply:
do you need gmp-ext? what is it good for?

Reply:
don know which of the comoser dependencies is using it, but whithout i got the following error when i try to `composer install` the freshly cloned repo:

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for mdanter/ecc v0.5.2 -> satisfiable by mdanter/ecc[v0.5.2].
    - mdanter/ecc v0.5.2 requires ext-gmp * -> the requested PHP extension gmp is missing from your system.
  Problem 2
    - mdanter/ecc v0.5.2 requires ext-gmp * -> the requested PHP extension gmp is missing from your system.
    - spomky-labs/jose v7.1.0 requires mdanter/ecc 0.5.* -> satisfiable by mdanter/ecc[v0.5.2].
    - Installation request for spomky-labs/jose v7.1.0 -> satisfiable by spomky-labs/jose[v7.1.0].

  To enable extensions, verify that they are enabled in your .ini files:
    - /usr/local/etc/php/php-cli.ini
    - /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
    - /usr/local/etc/php/conf.d/docker-php-ext-sodium.ini
    - /usr/local/etc/php/conf.d/docker-php-ext-zip.ini

Reply:
tracked it down

`laravel-notification-channels/webpush 3.0.1 Web Push Notifications driver for Laravel.
├──illuminate/notifications ^5.3
├──illuminate/support ^5.1
├──minishlink/web-push ^2.0
│  ├──guzzlehttp/guzzle ^6.2
│  │  ├──guzzlehttp/promises ^1.0
│  │  │  └──php >=5.5.0
│  │  ├──guzzlehttp/psr7 ^1.4
│  │  │  ├──php >=5.4.0
│  │  │  ├──psr/http-message ~1.0
│  │  │  │  └──php >=5.3.0
│  │  │  └──ralouphie/getallheaders ^2.0.5 || ^3.0.0
│  │  │     └──php >=5.6
│  │  └──php >=5.5
│  ├──lib-openssl *
│  ├──mdanter/ecc ^0.5.0
│  │  ├──ext-gmp *
│  │  ├──fgrosse/phpasn1 ^2.0
│  │  │  └──php >=7.0.0
│  │  └──php ^7.0
`